### ISSUE TITLE
Android 10 No longer supported? Repo issue?

### ISSUE THREAD

**Issue by SevenFactors** [Divested-Mobile/Mull-Fenix#237]
Hello,

I've been updating Mull from the official divestos f-droid repo for months.

F-droid reports no new compatible version of mull for Android 10. The latest and only version being shown on f-droid being version 127.

When I switch to f-droids owns repo, the latest version of Mull being shown is version 127.0.2

Is there something wrong with divestos f-droid repo OR is support for android 10 being dropped?

**Comment by SkewedZeppelin** [comment]
You need to set DivestOS.org as the preferred repo

Mull supports Android 5.0 and higher

**Comment by SevenFactors** [comment]
> You need to set DivestOS.org as the preferred repo
> 
> Mull supports Android 5.0 and higher

Yes. I've the official divestos repo selected. The whole reason I subscribed to it was to be able to get the latest divestos apps releases rather than waiting for f-droid servers to do their thing. I mentioned the fdroid repot to show how somehow the fdroid repo is displaying later releases than those being presented by the divestos repo. See attached images.

As you can see on 389, it reads: "this application is not compatible with the device. Do you want to install it anyway?"

The device is a moto e running android 10. 

Has the CPU requirement change or something?


![mull376](https://github.com/user-attachments/assets/ed1ea0d2-57ee-4ae8-ae8a-01a4f2b7d697)
![mull378](https://github.com/user-attachments/assets/63fa6efb-b526-4102-8b25-9f9d064751b4)
![mull379](https://github.com/user-attachments/assets/2582faec-5eca-48bc-8d93-54a71d341e6b)

**Comment by SkewedZeppelin** [comment]
Please try refresh your repos again.

Again, Mull has not dropped support for anything.

**Comment by SevenFactors** [comment]
> Please try refresh your repos again.
> 
> Again, Mull has not dropped support for anything.

I see. So the issue is somewhere else.  Manually refreshing doesn't do anything. I still get the same result for Mull.

Perhaps the issue if within the fdroid app itself. Perhaps there is something wrong with the repo connection or something. Maybe the latest fdroid store update messed something up.  I will keep troubleshooting.
